lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -101,6 +101,9 @@ public System.Threading.Tasks.Task RunTask()
                 {
                     try
                     {
+                        Logger.Log(Level.Error,
```

Comment:
Why is this in `Level.Error`? Wouldn't this code also be reached in normal termination of the task?